plugins/inputs/signalfx_metadata/signalfx-metadata.go

```diff
@@ -16,6 +16,8 @@ const pluginVersion = "0.0.30"
 var sampleConfig = `
   ## SignalFx metadata plugin reports metadata properties for the host
   ## Process List Collection Settings
+  ## disable process list collection
```

Comment:
Setting it to true disables it?  If you mean the fact that it is commented disables it that is kind of confusing.